doc/getting_started/getting_started_connectors.rst

```diff
@@ -4,11 +4,17 @@
 Connecting from your favorite language
 ================================================================================
 
-Now that you have a Tarantool database, let's see how to connect to it from
-Python, PHP and Go.
+In the :ref:`previous section <getting_started_db>`, you have learned how
```

Comment:
I would use a direct `:doc:` link here, because we refer to the whole document, not some heading in the middle.
https://www.tarantool.io/en/doc/latest/contributing/docs/markup/#linking-to-document

Reply:
В документе остался один абзац и оглавление, он теперь стал каким-то проходным. Зачем он вообще нам теперь? Может, вынести три документа про коннекторы на уровень вверх?

Reply:
Обсудили с @NickVolynkin: имеет смысл улучшить структуру всей главы https://www.tarantool.io/en/doc/latest/getting_started/ -- убрать многоуровневую структуру и вынести статьи по отдельным темам на уровень выше, написав соответствующие вводные абзацы как для индексной страницы, так и для статей по каждой из тем. Добавлю подзадачи в https://github.com/tarantool/doc/issues/1796. После их реализации вернемся к ревью данного PR.

Reply:
Решили сделать мёрдж как есть

> I would use a direct `:doc:` link here, because we refer to the whole document, not some heading in the middle.
> https://www.tarantool.io/en/doc/latest/contributing/docs/markup/#linking-to-document

Corrected.

Reply:
> Обсудили с @NickVolynkin: имеет смысл улучшить структуру всей главы https://www.tarantool.io/en/doc/latest/getting_started/ -- убрать многоуровневую структуру и вынести статьи по отдельным темам на уровень выше, написав соответствующие вводные абзацы как для индексной страницы, так и для статей по каждой из тем. Добавлю подзадачи в #1796. После их реализации вернемся к ревью данного PR.

Решили сделать мёрдж со сделанными изменениями из описания задачи, чтобы можно было в новую структуру опубликовать новый контент.

Вот эти пункты вынести в отдельную задачу:
- вынести статьи по отдельным темам (Using a Docker image; Using a package manager; Connecting from Python и т.д.) на уровень выше, т.е. убрать 2х-уровневую структуру. 
- Отредактировать в соответствии с этим вводные абзацы для индексной страницы и для статей по каждой из тем.